<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerGui</class>
 <widget class="QMainWindow" name="ServerGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ServerGui</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>611</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="faceTab">
     <attribute name="title">
      <string>Face Server</string>
     </attribute>
     <widget class="QGroupBox" name="groupBoxFaceBBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>321</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>Bounding Box Parameters</string>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>261</width>
         <height>21</height>
        </rect>
       </property>
       <property name="title">
        <string>Shape</string>
       </property>
       <widget class="QRadioButton" name="radioButtonFaceShapeCircle">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>0</y>
          <width>71</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Circle</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButtonFaceShapeRect">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>0</y>
          <width>101</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Rectangle</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="title">
        <string>Line Width</string>
       </property>
       <widget class="QSpinBox" name="spinBoxFaceLineWidth">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>0</y>
          <width>51</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>311</width>
         <height>51</height>
        </rect>
       </property>
       <property name="title">
        <string>Color</string>
       </property>
       <widget class="QRadioButton" name="radioButtonFaceColorRed">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>0</y>
          <width>61</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Red</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButtonFaceColorGreen">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>0</y>
          <width>71</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Green</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButtonFaceColorBlue">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>0</y>
          <width>61</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Blue</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButtonFaceColorBlack">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>30</y>
          <width>71</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Black</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButtonFaceColorWhite">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>30</y>
          <width>71</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>White</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>501</width>
        <height>251</height>
       </rect>
      </property>
      <property name="title">
       <string>Connection Information</string>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>51</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Status</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowserFaceStatus">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>40</y>
         <width>371</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>81</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Server Info</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowserFaceInfo">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>80</y>
         <width>371</width>
         <height>151</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="objectTab">
     <attribute name="title">
      <string>Object Server</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>501</width>
        <height>251</height>
       </rect>
      </property>
      <property name="title">
       <string>Connection Information</string>
      </property>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>51</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Status</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowserObjectStatus">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>40</y>
         <width>371</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>81</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Server Info</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowserObjectInfo">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>80</y>
         <width>371</width>
         <height>151</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBoxFaceBBox_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>321</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>Bounding Box Parameters</string>
      </property>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>261</width>
         <height>21</height>
        </rect>
       </property>
       <property name="title">
        <string>Shape</string>
       </property>
       <widget class="QRadioButton" name="radioButtonObjectShapeCircle">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>0</y>
          <width>71</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Circle</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButtonObjectShapeRect">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>0</y>
          <width>101</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Rectangle</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="title">
        <string>Line Width</string>
       </property>
       <widget class="QSpinBox" name="spinBoxObjectLineWidth">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>0</y>
          <width>51</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>311</width>
         <height>51</height>
        </rect>
       </property>
       <property name="title">
        <string>Color</string>
       </property>
       <widget class="QRadioButton" name="radioButtonObjectColorRed">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>0</y>
          <width>61</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Red</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButtonObjectColorGreen">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>0</y>
          <width>71</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Green</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButtonObjectColorBlue">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>0</y>
          <width>61</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Blue</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButtonObjectColorBlack">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>30</y>
          <width>71</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Black</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButtonObjectColorWhite">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>30</y>
          <width>71</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>White</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="sportsTab">
     <attribute name="title">
      <string>Sports Server</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>611</width>
        <height>271</height>
       </rect>
      </property>
      <property name="title">
       <string>Match Information</string>
      </property>
      <widget class="QGraphicsView" name="graphicsViewSportsLogo1">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>100</y>
         <width>143</width>
         <height>153</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditSportsTeam2">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>60</y>
         <width>151</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBoxSportsTeam2">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>60</y>
         <width>61</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>51</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Team 1</string>
       </property>
      </widget>
      <widget class="QGraphicsView" name="graphicsViewSportsLogo2">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>100</y>
         <width>143</width>
         <height>153</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditSportsTeam1">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>60</y>
         <width>151</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBoxSportsTeam1">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>60</y>
         <width>61</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>60</y>
         <width>51</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Team 2</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>150</y>
         <width>51</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Vs</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>30</y>
         <width>41</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Score</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>501</width>
        <height>251</height>
       </rect>
      </property>
      <property name="title">
       <string>Connection Information</string>
      </property>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>51</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Status</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowserSportsStatus">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>40</y>
         <width>371</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>81</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Server Info</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowserSportsInfo">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>80</y>
         <width>371</width>
         <height>151</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuVios_Server">
    <property name="title">
     <string>Vios Server</string>
    </property>
   </widget>
   <addaction name="menuVios_Server"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
